src/Tools/ExternalAccess/OmniSharp/ExtractClass/IOmniSharpExtractClassOptionsService.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.ExtractClass
 {
     internal interface IOmniSharpExtractClassOptionsService
     {
-        Task<OmniSharpExtractClassOptions?> GetExtractClassOptionsAsync(Document document, INamedTypeSymbol originalType, ISymbol? selectedMember);
+        Task<OmniSharpExtractClassOptions?> GetExtractClassOptionsAsync(Document document, INamedTypeSymbol originalType, ImmutableArray<ISymbol> selectedMembers);
```

Comment:
i don't think this is safe to do.  this will break omnisharp.  Right @333fred ?  

happy to explain more offline.

Reply:
See Fred's comments earlier in this thread

Reply:
n/m. @Soreloser2 mentioned you prefer this approach @333fred

Reply:
Yup. O# is already converting a list to a single member for this, it will simplify the code.